cairo/src/runner.rs

```diff
@@ -3,9 +3,12 @@
 
 use crate::flags::*;
 use crate::memory::CairoMemory;
-use crate::word::{CairoWord, Decomposition};
+use crate::word::{CairoWord, FlagBits, FlagSets, Offsets};
 use ark_ff::Field;
 
+/// Number of memory accesses per instruction
+pub const ACC_PER_INS: usize = 4;
```

Comment:
where is this used?

Reply:
Actually, this is being used only in the memory constraints, but I must have missed to remove this one from this PR.